tests/test_modeling_common.py

```diff
@@ -3046,6 +3046,7 @@ def test_multi_gpu_data_parallel_forward(self):
             with torch.no_grad():
                 _ = model(**self._prepare_for_class(inputs_dict, model_class))
 
+    @require_torch_gpu
```

Comment:
`@require_torch_multi_gpu` on the next line was previously made non-CUDA specific by https://github.com/huggingface/transformers/commit/11c27dd331151e7d2ac20016cce11d9d7c4b1756. Right now it sets requirement for having any multi-GPUs on the system.

I wonder, maybe it makes sense to rename `@require_torch_gpu` to `@require_torch_cuda` to avoid naming collision? I can follow up on that in separate PR.

Reply:
cc @ydshieh on this! 🤗